### DISCUSSION TITLE
Zipping with 'store' method must do only 1 pass

### DISCUSSION THREAD

**Discussion by Alexey-T** [doublecmd/doublecmd#1203] [General]
Make 3Gb file with python script
```
f=open('/home/user/big.txt', 'w')
for i in range(3*1024*1024*1024//100):
    f.write('e'*99+'\n')
del f
```

now pack it to .zip file with 'store' zip method.
- DC makes 2 passes! and it is very slow. must be 1 pass.
-  maybe each pass is 2-3 times slower than copying of file, but with 'store' method it must be fast. 

 DC 1.0.11~272. Xubuntu 22.

**Comment by alexx2000** [comment]
Why are you think that DC makes 2 passes? I tested it and see only one pass. Data copied with very small buffer (8KB), maybe it is a cause of slow copy.

**Comment by Alexey-T** [comment]
because I see window with 2 progressbars. 

stage 1  - top progressbar fills, very slow.
stage 2 - bottom progressbar fills, also slow.
